finished section

King-Franch opened the pull request:
створення скелету секцяї "new watches" і написання стилів для адаптиву під 3 пристрою: телефон, планшет, десктоп

King-Franch committed:
finished section

створення скелету секцяї "new watches" і написання стилів для адаптиву під 3 пристрою: телефон, планшет, десктоп

ebikbaev reviewed (changes requested):
ошибки указаны в трело

King-Franch committed:
едіт

- добавлені падінги на мобільній версії в 5пх.
- центрування секцій. Назва секції і діскріпшн більше не прилипають до стінки.
- на десктоп версії маржини замінені на падінги.
- добавлений падінг для нижньої секції

ArtemVasiletc reviewed (approved):
for 320px 
.container {
  padding-left: 5px;
  padding-right: 5px;
{

@media screen and (min-width: 375px) {
  .container {
    max-width: 375px;
    padding-left: 20px;
    padding-right: 20px;
  }
}

@media screen and (min-width: 1280px) {
.container {
    padding-left: 52px;
    padding-right: 52px;
}
